### DISCUSSION TITLE
User needs to renew their server every 7 days to keep the server

### DISCUSSION THREAD

**Discussion by 1gzify** [Paymenter/Paymenter#215] [Ideas]
make it so User needs to renew their server every 7 days to keep the server otherwise it will be deleted

**Comment by finnjosten** [comment]
Why? If they pay for it, it should stay. And for free servers I still don't see why you would want this

**Reply by 1gzify** [reply]
No, i meant for free servers. i offer free hosting. for the paid hosting they keep it for how much they paid.

**Reply by ProgyNL** [reply]
You mean like there is an trail product you can setup when it is almost time to delete, you can cancel the product or make an payment to keep it?

**Comment by 1gzify** [comment]
> You mean like there is an trail product you can setup when it is almost time to delete, you can cancel the product or make an payment to keep it?

That too @ProgyNL. But i meant like for example i offer free hosting and people can order for free in paymenter. To prevent people just getting servers. They will have to renew their server every 7 - 10 days to keep it otherwise it will be deleted to open space for other customers.

**Comment by finnjosten** [comment]
Who would ever want to go with free hosting, your just losing money
